services/userService.js

```diff
@@ -44,8 +43,52 @@ const mailCheck = async (email) => {
     return true;
   }
 };
+//토큰관련
+const getUserById = async (email) => {
+  return await userDao.getUserId(email);
+};
+//마이페이지 정보전송
+const mypage = async (email) => {
+  return await userDao.getUser(email);
+};
```

Comment:
```suggestion
const getUserInfo = async (userId) => {
  return userDao.getUserInfo(userId);
};
```
1. 주석 지워주세요.
2. 동기적으로 실행되야하는 코드가 아니기 때문에 await는 빼주세요.
3. 모든 코드들에 위의 내용을 적용해주세요.

Reply:
네 확인하고 전부 적용했습니다 !